frontend/packages/client/src/components/ProposalCreate/StepOne/index.js

```diff
@@ -8,17 +8,37 @@ import React, {
 import { Editor } from "react-draft-wysiwyg";
 import {
   EditorState,
-  ContentBlock,
   AtomicBlockUtils,
   Modifier,
+  ContentState,
+  DefaultDraftBlockRenderMap,
+  SelectionState,
+  RichUtils,
 } from "draft-js";
+import { Map } from "immutable";
```

Comment:
Is there a reason for using `Map` from immutable instead of using the native Map? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Reply:
Was looking into this further and looks like `draft-js` is using Immutable so we can't use the native map. After looking at our yarn.lock file we already have immutable package. If we update the package.json file to use: `immutable@^4.0.0` this will prevent us from having 3 different version of immutable included. Currently immutable doesn't support tree shaking so the entire package is being included so by pinning to v4.0 we can reduce the bundle size